src/index.html

```diff
@@ -11,12 +11,100 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- Перша навігаційна панель -->
+    <header>
+      <div class="logo-container">
+        <a
+          class="imgLogo logo-1200"
+          href="index.html"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link is-active"
+            >
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol if they are meant to be internal page links.